applications/CoSimulationApplication/tests/dem_fem_cable_net/MaterialsDEM.json

```diff
@@ -23,6 +23,7 @@
             "COEFFICIENT_OF_RESTITUTION": 0.2,
             "STATIC_FRICTION": 0.9999999999999999,
             "DYNAMIC_FRICTION": 0.9999999999999999,
+            "DEM_ROLLING_FRICTION_MODEL_NAME": "DEMRollingFrictionModelConstantTorque",
```

Comment:
Can you please remove this here and make it a default?
Otherwise every existing case is broken

Reply:
Yes, I can. I have tested that making this parameter a default can works well. But listing it in the input file maybe more clear because we will have several different rolling friction models in the future. What is your opinion about this? @maceligueta 
I have also tested that adding this parameter in the input file can works well with the existing cases.

Reply:
If the case runs fine without this line (because the default value is ok) I would remove that from the CoSimulation application. However, in the DEM and SwimmingDEM app I would like to see which model is being used, so I would keep this line in DEM and SDEM.

Reply:
Okay. I will make it a default parameter and print some information in screen to tell user that they are using a default rolling friction model.